README.md

```diff
@@ -161,10 +161,9 @@ Follow the instruction on <https://rustup.rs/> to install rust and cargo.
 
 Then try running `npm install` again.
 
-Make sure that your nodejs version is `16.0.0` or newer.
```

Comment:
Node v16 has reached end of life

Reply:
electron 22 (which we use) still uses node 16: https://www.electronjs.org/blog/electron-22-0

Reply:
so I think it's better to do that later as part of #3381 also to really change it we need to update all occurrences and ci files.

Reply:
> electron 22 (which we use) still uses node 16: https://www.electronjs.org/blog/electron-22-0

This is really unrelated imo

> so I think it's better to do that later as part of #3381 also to really change it we need to update all occurrences and ci files.

Much better argument! I'll tweak this and make a separate PR for all node 16 thingies :smile:

Reply:
> This is really unrelated imo

not really, the version electron ships is the one the code actually runs with.

Reply:
> > This is really unrelated imo
> 
> not really, the version electron ships is the one the code actually runs with.

So then it doesn't matter what node version you're running.